widget/ui/src/components/ListItem/ListItem.styles.ts

```diff
@@ -50,13 +50,15 @@ export const BaseListItem = styled('li', {
     hasDivider: {
       true: {
         borderBottom: '1px solid',
-        $$color: '$colors$neutral300',
+        borderColor: '$neutral300',
         [`.${darkTheme} &`]: {
-          $$color: '$colors$neutral400',
+          borderColor: '$neutral400',
         },
-        borderColor: '$$color',
         borderBottomRightRadius: 0,
         borderBottomLeftRadius: 0,
+        '&:hover': {
+          borderRadius: '$xs',
+        },
```

Comment:
The top and bottom border radius don't match each other :
![image02](https://github.com/rango-exchange/rango-client/assets/120931880/b9d0b2af-c8e7-4df9-9ae6-176d77a46af6)

Reply:
It's matched with Figma. Let's talk about it if you have any comments.